loan/src/main/java/com/iluwatar/loan/Main.java

```diff
@@ -0,0 +1,54 @@
+package com.iluwatar.loan;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
```

Comment:
Here, above the main class (usually named App.java), please explain the pattern briefly. Describe how the code example implements the pattern.